pinot-common/src/main/java/org/apache/pinot/common/request/PinotQuery.java

```diff
@@ -277,6 +277,7 @@ public void clear() {
     this.groupByList = null;
     this.orderByList = null;
     this.havingExpression = null;
+    // if instance gets cleared and reused make sure default limit is applied (if set)
```

Comment:
This is auto-generated from `query.thrift`. So this file shouldnt be edited ?

Reply:
You're right, it'll get overwritten on next re-generation so I removed the comment.